doc/release-notes-1.3.1.rst

```diff
@@ -0,0 +1,64 @@
+.. _ncs_release_notes_131:
+
+|NCS| v1.3.1 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.1 for prototyping and evaluation.
+Support for production and deployment in end products is coming soon.
+
+
+Highlights
+**********
+
+* LwM2M carrier library v0.9.1 is certified with AT&T.
+* Bluetooth Host is now Bluetooth Core Specification v5.1 qualified.
+* nRF Bluetooth LE Controller (in sdk-nrfxlib) is now Bluetooth Core Specification v5.2 qualified.
```

Comment:
It is now renamed as softdevice_controller, should not this stated in highlights @jdstapes / @bjorn-tore-taraldsen

Reply:
It has been renamed on master, but it's still nRF Bluetooth LE Controller on the v1.3 branch.

Reply:
OK